lib/block_view_validator_test.go

```diff
@@ -163,7 +163,9 @@ func _testValidatorRegistration(t *testing.T, flushToDB bool) {
 		require.Contains(t, err.Error(), RuleErrorValidatorMissingVotingPublicKey)
 	}
 	{
-		// RuleErrorValidatorInvalidVotingSignatureBlockHeight: missing
+		// Got rid of VotingSignatureBlockHeight but going to leave this test here.
```

Comment:
nit: IMO we should also delete this test.

Reply:
+1

Reply:
Addressed in https://github.com/deso-protocol/core/pull/531.